Source/IGListStackedSectionController.m

```diff
@@ -187,6 +187,18 @@ - (UICollectionViewCell *)cellForItemAtIndex:(NSInteger)index sectionController:
     return cells;
 }
 
+- (NSArray<NSIndexPath *> *)visiblePathsForSectionController:(IGListSectionController<IGListSectionType> *)sectionController {
+    NSMutableArray *paths = [NSMutableArray new];
+    id<IGListCollectionContext> collectionContext = self.collectionContext;
+    NSArray *visiblePaths = [collectionContext visiblePathsForSectionController:self];
+    for (NSIndexPath *path in visiblePaths) {
+        if (self.sectionControllersForItems[path.section] == sectionController) {
```

Comment:
Hmm I think this should actually be `path.item` instead. A unit test would help to confirm.